.github/infrastructure/terraform/conformance/pubsub/aws/snssqs/snssqs.tf

```diff
@@ -0,0 +1,94 @@
+resource "aws_sns_topic" "testTopic" {
```

Comment:
Suggestion - It's customary in Terraform to add the provider block similar to the following.
This could be added to the top of this file or as a separate file usually called **versions.tf**
```terraform
terraform {
  required_version = ">=0.13"

  required_providers {
    aws = {
      source  = "hashicorp/aws"
      version = "~> 4.0"
    }
  }
}

provider "aws" {
}
```

Reply:
Sure I can add this for clarity.